app.js

```diff
@@ -836,6 +838,20 @@ define(function(require) {
 						}
 					);
 				},
+				function(_data, waterfallCallback) {
+					self.createVMBox(data.vmbox,
```

Comment:
Basically move it to this handle to something like:
```suggestion
          data.vmbox.owner_id = data.user.id
					self.createVMBox(data.vmbox,
```

Reply:
Done. Thanks!